client/src/components/ListingsGroup.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, Row } from "react-bootstrap";
+// todo: import { Listing } from '';
+
+function ListingsGroup({ content, createdAt, id }) {
```

Comment:
I'm a bit confuse for this function. Do you want to return all the listings, or a specific listing? What is Listings in const [listings, setListings] = useState([{}, {}, {}]);, an array, 3 empty objects?

Reply:
{ content, createdAt, id } where is this coming from?